modules/backend/widgets/Lists.php

```diff
@@ -1544,6 +1544,13 @@ protected function getSortColumn()
         return $this->sortColumn;
     }
 
+    /*
+     * Returns the current sort direction or default of 'asc'
+     */
+    public function getSortDirection () {
+        return $this->sortDirection = $this->sortDirection ?: 'asc';
```

Comment:
@empower-josh This could be simplified to a null coalesce operation.
```suggestion
        return $this->sortDirection ?? 'asc';
```